src/main.py

```diff
@@ -237,6 +241,54 @@ async def info(interaction: Interaction) -> None:
     await bc.send_stats(interaction)
 
 
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(description="The file to submit."),
+    submit_message: Optional[str] = SlashOption(
+        required=False, description="The submission message text."
+    ),
+) -> None:
+    """Show a preview of a submission's "Now Playing" embed."""
+    user = interaction.user
+
+    if not discord_utils.is_valid_media(uploaded_file.content_type):
+        await interaction.response.send_message(
+            "You didn't send a valid media type. \nTry again.",
```

Comment:
Nitpick, but space and then newline is a little weird. I'd probably prefer "You didn't send a valid media type, try again." anyways.